lib/handlers/addCategory.ts

```diff
@@ -0,0 +1,19 @@
+import { CategoryService } from '../services/categoryService';
+import { CategoryRepository } from '../repositories/categoryRepository';
+import { Category } from "../models/category";
+
+interface AppSyncEvent {
+    arguments: {
+        [key: string]: any;
+    };
+}
```

Comment:
What's the point of having `AppSyncEvent` in every handler & then make it kinda useless with `[key: string]: any;`. We expect a certain type here, we should have narrow this to that type.

Reply:
taken care for all methods of category, product, and orders